src/Angor/Avalonia/Angor.Contexts.Funding/Founder/Operations/GetInvestments.cs

```diff
@@ -107,18 +166,14 @@ IObservable<DirectMessage> InvestmentMessagesObs()
             }
         }
 
-        private long GetAmount(string transactionHex)
+        private static long GetAmount(Transaction transaction)
         {
-            var investorTrx = networkConfiguration.GetNetwork().CreateTransaction(transactionHex);
-
-            return investorTrx.Outputs.AsIndexedOutputs()
+            return transaction.Outputs.AsIndexedOutputs()
                 .Skip(2)
-                .Take(investorTrx.Outputs.Count - 3)
+                .Take(transaction.Outputs.Count - 3)
                 .Sum(x => x.TxOut.Value.Satoshi);
```

Comment:
this looks wrong but I guess you can fix it later

Reply:
Oh, I needed to copy that code. I'll check with @DavidGershony

Reply:
should be take count - 1 assuming there is a change address, but really we should pass in the number of stages and take that.  you should have it in the project info you get at the top.

Reply:
OK; so I just passed the project to the GetAmount method. Is this what you mean?

```
 private static long GetAmount(Transaction transaction, Project project)
        {
            return transaction.Outputs.AsIndexedOutputs()
                .Skip(2)
                .Take(project.Stages.Count())
                .Sum(x => x.TxOut.Value.Satoshi);
        }
```